zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -1088,7 +1088,10 @@ private InterpreterSetting getDefaultInterpreterSetting(List<InterpreterSetting>
   public InterpreterSetting getDefaultInterpreterSetting(String noteId) {
     return getDefaultInterpreterSetting(getInterpreterSettings(noteId));
   }
-
+  
+  public boolean isBinding(String noteId, String replName) {
```

Comment:
@bzz I want to ask your opinion. If I make `Note#isBinding` method, I remove `InterpreterFactory#isBinded` method. Do you agree with me?